packages/discord.js/src/util/Util.ts

```diff
@@ -18,34 +16,41 @@ class Util extends null {
    * @param {...Object<string, boolean|string>} [props] Specific properties to include/exclude.
    * @returns {Object}
    */
-  static flatten(obj, ...props) {
+  static flatten(obj: object, ...props: Iterable<[string, boolean | string]>[]): unknown {
```

Comment:
It’s good practice to use unknown in this specific situation 
```suggestion
  static flatten(obj: unknown, ...props: Iterable<[string, boolean | string]>[]): unknown {
```

Reply:
`unknown` is a wider type than `object`

Reply:
Exactly, and this function accepts any kind of input as seen by the check in the line right below this one, so this should be changed